src/components/Hero/index.tsx

```diff
@@ -1,34 +1,83 @@
 import React from 'react';
 import { Link } from 'gatsby';
 
+import { detectOS, UserOS } from '../../util/detectOS';
+import { useReleaseHistory, ReleaseData } from '../../hooks/useReleaseHistory';
+
 interface Props {
   title: string;
   subTitle: string;
 }
 
-const NodeVersion = 'Version 10.15.3';
-
-const Hero = ({ title, subTitle }: Props): JSX.Element => (
-  <div className="home-page-hero">
-    <h1>{title}</h1>
-    <h2 className="sub-title t-subheading">{subTitle}</h2>
-    <div className="btn-ctas">
-      <div className="download-lts-container">
-        <button className="download-lts-cta t-body1" type="button">
-          Download Node (LTS)
-        </button>
-        <p className="links t-caption">
-          {NodeVersion} - <Link to="/download">What’s new</Link> /{' '}
-          <Link to="/download">Get Current</Link>
-        </p>
+function downloadUrlByOS(userOS: UserOS, version: string): string {
+  const baseURL = `https://nodejs.org/dist/${version}`;
+
+  if (userOS === UserOS.MOBILE) {
+    return baseURL;
+  }
+
+  if (userOS === UserOS.MAC) {
+    return `${baseURL}/node-${version}.pkg`;
+  }
+
+  if (userOS === UserOS.WIN) {
+    if (
+      navigator.appVersion.indexOf('WOW64') !== -1 ||
```

Comment:
Which OS is this? I don't see it listed in the UserOS util

Reply:
It is [Windows 32-bit on Windows 64-bit](https://en.wikipedia.org/wiki/WoW64). I added to verify if it is a 64 bits OS to download `node-${version}-x64.msi` or `node-${version}-x86.msi` window installer

Reply:
cool I didn't know that was a thing. I also found this. https://stackoverflow.com/questions/1741933/detect-64-bit-or-32-bit-windows-from-user-agent-or-javascript

Reply:
yea! I got exactly from there the `WOW64` validation :D